types/coin.go

```diff
@@ -881,7 +881,7 @@ func ParseCoinNormalized(coinStr string) (coin Coin, err error) {
 		return Coin{}, err
 	}
 
-	coin, _ = NormalizeDecCoin(decCoin).TruncateDecimal()
+	coin, _ = normalizeDecCoin(decCoin).TruncateDecimal()
```

Comment:
@tac0turtle I understand the prior code did it, but given that the signature for this function is (Coin, error) we should return the error so please do: return normalizeDecCoin(decCoin).TruncateDecimal() instead of ignoring the error then coercing a nil error, which is double work.

Reply:
the second item here isnt an error, confusing func signature

Reply:
Gotcha and thanks!